spec.md

```diff
@@ -1835,6 +1835,8 @@ module MyModule :
    out[4] <= in
 ```
 
+Bit-indexing is not supported on memory ports.
```

Comment:
Why is this necessary? The only memory port field that would be affected is the `data` field and for that we could always treat it like a memory of a bundle type.

Reply:
Yes you're right the operation should be allowed on memory ports, given that the ports are fully initialized just as with bundles.